Remove warning "WARN[0000] Found default OCIruntime /nix/store/.../bin/crun path which is missing from [engine.runtimes] in containers.conf

It doesn't make sense as we promote using the podman wrapper where runtime paths will vary because they are nix store paths.
---
 vendor/github.com/containers/common/pkg/config/config.go | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/vendor/github.com/containers/common/pkg/config/config.go b/vendor/github.com/containers/common/pkg/config/config.go
index 4a98c7e92..4a95a2a49 100644
--- a/vendor/github.com/containers/common/pkg/config/config.go
+++ b/vendor/github.com/containers/common/pkg/config/config.go
@@ -605,8 +605,7 @@ func (c *EngineConfig) findRuntime() string {
 				return name
 			}
 		}
-		if path, err := exec.LookPath(name); err == nil {
-			logrus.Warningf("Found default OCIruntime %s path which is missing from [engine.runtimes] in containers.conf", path)
+		if _, err := exec.LookPath(name); err == nil {
 			return name
 		}
 	}
--
2.30.0
